### PR TITLE
feat-4175-node_example_send_http_request

### PR DESCRIPTION
**Pull request by omjikush09**
Tested the function by running, using appwrite CLI and docker container and it worked fine

### CODE REVIEW

**Comment by omjikush09** [comment]
Issue  https://github.com/appwrite/appwrite/issues/4175

File: node/send_http_request/README.md
Comment (Meldiron) [inline node/send_http_request/README.md:1]:
```suggestion
# 📱 Send Http Request
```

File: node/send_http_request/README.md
Comment (Meldiron) [inline node/send_http_request/README.md:3]: Let's fix the description

File: node/send_http_request/README.md
Comment (Meldiron) [inline node/send_http_request/README.md:21]:
```suggestion
{
    "success":true,
    "response":{"headers":{},"code":200,"body":"{\"total\":27,\"countries\":[]}"}
}
```

File: node/send_http_request/index.js
Comment (Meldiron) [inline node/send_http_request/index.js:13]: Let's use `res.json()` with success:false instead of throwing error for consistency.

File: node/send_http_request/index.js
Comment (Meldiron) [inline node/send_http_request/index.js:35]: Let's add `console.log` here just like in try/catch earlier, for consistency.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Great PR! 🤩 Requested a few changes. Once addressed, I can approve this PR.

File: node/send_http_request/index.js
Comment (omjikush09) [reply node/send_http_request/index.js:35]: As I asked Steven, He said to remove the `cosole.log` because it was causing issues in running (with docker not with appwrite CLI). So I have removed all `console.log`  now

File: node/send_http_request/index.js
Comment (Meldiron) [inline node/send_http_request/index.js:13]: Let's update the error message here. Could be something along likes `Payload does not have a proper structure.`.

Comment (omjikush09) [reply node/send_http_request/index.js]: Done with changes

**Comment by Meldiron** [comment]
@omjikush09 Looks almost perfect 😇 There is one more unresolved comment by me regarding the description in the readme. It says it `figures out country in which a phone number`. We need to change it to a new description talking about this function sending a HTTP request.

**Comment by omjikush09** [comment]
@Meldiron, Hopefully now you can merge.

**Review by Meldiron — APPROVED** [review]
Loooooooks great 😍

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Comment by stnguyen90** [comment]
Closes: https://github.com/appwrite/appwrite/issues/4175

**Pull request merged**
